src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/transforms/charts/metric.ts

```diff
@@ -118,17 +149,39 @@ function buildVisualizationState(config: MetricState): MetricVisualizationState
   };
 }
 
+function getCompareToApi(
```

Comment:
shouldn't we use always the same verbs? `compareCompareToAPI`? we should not care if `compare` appears 2 times, but we should use the same structure `verb + resource + ToWhat`